Gemfile

```diff
@@ -1,18 +1,13 @@
 source "http://rubygems.org"
 
-
 group :development, :test do
-  gem "rails", ">=4.0.0"
-  gem "sqlite3", "1.3.8"
```

Comment:
Removed version constraint because it wasn't working with ruby 2
```
Gem::Ext::BuildError: ERROR: Failed to build gem native extension.
    /home/travis/.rvm/rubies/ruby-2.2.0/bin/ruby -r ./siteconf20150917-2405-1qvxxiv.rb extconf.rb 
checking for sqlite3.h... yes
checking for sqlite3_libversion_number() in -lsqlite3... yes
checking for rb_proc_arity()... yes
checking for sqlite3_initialize()... yes
checking for sqlite3_backup_init()... yes
checking for sqlite3_column_database_name()... yes
checking for sqlite3_enable_load_extension()... yes
checking for sqlite3_load_extension()... yes
checking for sqlite3_open_v2()... yes
checking for sqlite3_prepare_v2()... yes
checking for sqlite3_int64 in sqlite3.h... yes
checking for sqlite3_uint64 in sqlite3.h... yes
creating Makefile
make "DESTDIR=" clean
make "DESTDIR="
compiling statement.c
statement.c: In function ‘bind_param’:
statement.c:261:7: warning: implicit declaration of function ‘RBIGNUM’ [-Wimplicit-function-declaration]
statement.c:261:11: error: invalid type argument of ‘->’ (have ‘int’)
statement.c:261:32: error: ‘SIZEOF_BDIGITS’ undeclared (first use in this function)
statement.c:261:32: note: each undeclared identifier is reported only once for each function it appears in
statement.c: In function ‘reset_bang’:
statement.c:293:7: warning: variable ‘status’ set but not used [-Wunused-but-set-variable]
statement.c: In function ‘clear_bindings’:
statement.c:313:7: warning: variable ‘status’ set but not used [-Wunused-but-set-variable]
make: *** [statement.o] Error 1
make failed, exit code 2
Gem files will remain installed in /home/travis/.rvm/gems/ruby-2.2.0/gems/sqlite3-1.3.8 for inspection.
Results logged to /home/travis/.rvm/gems/ruby-2.2.0/extensions/x86_64-linux/2.2.0/sqlite3-1.3.8/gem_make.out
Installing mail 2.6.3
An error occurred while installing sqlite3 (1.3.8), and Bundler cannot continue.
Make sure that `gem install sqlite3 -v '1.3.8'` succeeds before bundling.
```